apps/email/js/cards/message_list.html

```diff
@@ -102,7 +102,7 @@ <h1 data-prop="headerNode" class="msg-listedit-header-label">
       </p>
     </form>
     <div data-prop="messagesContainer" class="msg-messages-container"
-         role="listbox">
+         role="listbox" aria-multiselectable="true">
```

Comment:
Apologies for my aria newness, but is this OK to always set on this div, even when the card is not in edit mode, and is not really multi-select, but more "tap on item and goes to a new card"?